packages/amplify/amplify_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+## 1.0.0-next.1
+
+### Breaking Changes
+
+Three new categories have added support for Web + Desktop: API, Storage, and Analytics!
+
+See our [docs](https://docs.amplify.aws/lib/q/platform/flutter/) for guides on how to migrate to these new versions.
+
+### Features
+- feat(api): GraphQL Custom Request Headers ([#1938](https://github.com/aws-amplify/amplify-flutter/pull/1938))
+- feat(api): Subscription Reconnection ([#2074](https://github.com/aws-amplify/amplify-flutter/pull/2074))
+- feat(api): authorizationMode property for GraphQLRequest ([#2143](https://github.com/aws-amplify/amplify-flutter/pull/2143))
```

Comment:
nit/suggestion: Would it make sense to add a note about REST API interface changes? For appropriate PR link, I guess you would have to use https://github.com/aws-amplify/amplify-flutter/pull/2148.

Reply:
Done 👍